basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRequestHandler.java

```diff
@@ -18,14 +18,14 @@
     @Autowired
     protected HandlerFactory<RequestHandler> handlerFactory;
 
-    private Class<T> clazz;
+    private static Class clazz;
 
     @Override
     public Response<?> handle(Request<?> msg) {
-        clazz = (Class<T>) ((ParameterizedType) this.getClass().getGenericSuperclass()).getActualTypeArguments()[0];
+        clazz = (Class) ((ParameterizedType) this.getClass().getGenericSuperclass()).getActualTypeArguments()[0];

```

Comment:
As I see, we cant store this in one static variable, cause we need different types, as LoginRequest, or ContactSyncRequest. And static even cant be initialized this way without actual object.